content/en/Integrations/IntegrationBuilder/How to Guides/github.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "GitHub"
+linkTitle: "GitHub"
+weight: 40
+---
+
+{{% pageinfo %}}
+How to use Integration Builder to push findings to GitHub Issues.
+{{% /pageinfo %}}
+
+## Push findings to GitHub Issues
+
+You can use Integration Builder to push pentest and/or DAST findings to GitHub Issues.
+
+### Prerequisites
+
+To use GitHub recipes, you must have a GitHub account and a repository where you
+want to push findings as issues.
+
+{{% expand "GitHub Documentation" %}}
+- [Create a GitHub account](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github)
+- [Create a repository](https://docs.github.com/en/get-started/start-your-journey/hello-world#step-1-create-a-repository)
+{{% /expand %}}
+
+## Set up the GitHub connection
+
+1. Click the create connection button
```

Comment:
It's probably worth mentioning that they need to navigate to the projects tab first.